bin/foreman-proxy-certs-generate

```diff
@@ -20,20 +20,33 @@ class ForemanProxyCertsGenerate
 
   private
 
+  # rubocop: disable Metrics/MethodLength
   def generate_answers_file
-    temp_file('foreman-proxy-certs-answer',
-              <<-EOF
+    if File.file?('/etc/foreman-installer/scenarios.d/last_scenario.yaml')
+      scenario = YAML.load_file('/etc/foreman-installer/scenarios.d/last_scenario.yaml').fetch(:answer_file)
+      organization = begin
+                       YAML.load_file("#{scenario}")['foreman']['initial_organization']
+                     rescue
+                       nil
+                     end
+    else
+      organization = "Default Organization"
+      if organization.nil?
+        organization = "Default Organization"
+      end
+      temp_file('foreman-proxy-certs-answer',
```

Comment:
I think this isn't in the right place, it's part of the else block that begins on line 32.  The way the code is written, you'll only ever create the temp answers file if there's no last scenario.